internal/rootcoord/meta_table.go

```diff
@@ -936,6 +936,54 @@ func (mt *MetaTable) DropIndex(collName, fieldName, indexName string) (typeutil.
 	return dropIdxID, true, nil
 }
 
+func (mt *MetaTable) GetInitBuildIDs(collName, indexName string) ([]UniqueID, error) {
+	mt.ddLock.RLock()
+	defer mt.ddLock.RUnlock()
+
+	collID, ok := mt.collName2ID[collName]
+	if !ok {
+		collID, ok = mt.collAlias2ID[collName]
+		if !ok {
+			return nil, fmt.Errorf("collection not exist with name = %s", collName)
+		}
+	}
+	collMeta, ok := mt.collID2Meta[collID]
+	if !ok {
+		return nil, fmt.Errorf("collection not exist with name = %s", collName)
+	}
+
+	var indexID typeutil.UniqueID
+	for _, info := range collMeta.FieldIndexes {
+		idxMeta, ok := mt.indexID2Meta[info.IndexID]
+		if !ok {
+			continue
+		}
+		if idxMeta.IndexName != indexName {
+			continue
+		}
+		indexID = info.IndexID
+		break
+	}
+
+	if indexID == 0 {
+		log.Warn("get init buildIDs, index not found", zap.String("collection name", collName),
+			zap.String("index name", indexName))
+		return nil, fmt.Errorf("index not found with name = %s", indexName)
```

Comment:
Better to also add collection name in error.

Reply:
ok